classes/class-ppc-pagesetups.php

```diff
@@ -177,9 +177,7 @@ public function ppc_custom_box_html() {
 						<input type="checkbox" name="checkbox[]" class="ppc_checkboxes" value= "<?php echo esc_attr( $ppc_key ); ?>"
 					<?php
 					if ( ! empty( $value ) ) {
```

Comment:
We can skip this isset check as well because even if the variable is empty `checked()` will not echo markup for the field.